README.md

```diff
@@ -1,21 +1,20 @@
 <h1 align="center">package.json validator</h1>
 
-<p align="center">Tools to validate <code>package.json</code> files.</p>
+<p align="center">Tools to validate package.json files.</p>
 
 <p align="center">
+	<!-- prettier-ignore-start -->
+	<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+	<a href="#contributors" target="_blank"><img alt="👪 All Contributors: 32" src="https://img.shields.io/badge/%F0%9F%91%AA_all_contributors-32-21bb42.svg" /></a>
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+	<!-- prettier-ignore-end -->
 	<a href="https://github.com/JoshuaKGoldberg/package-json-validator/blob/main/.github/CODE_OF_CONDUCT.md" target="_blank"><img alt="🤝 Code of Conduct: Kept" src="https://img.shields.io/badge/%F0%9F%A4%9D_code_of_conduct-kept-21bb42" /></a>
-	<a href="https://github.com/JoshuaKGoldberg/package-json-validator/blob/main/LICENSE.md" target="_blank"><img alt="📝 License: MIT" src="https://img.shields.io/badge/%F0%9F%93%9D_license-MIT-21bb42.svg"></a>
-	<a href="http://npmjs.com/package/package-json-validator"><img alt="📦 npm version" src="https://img.shields.io/npm/v/package-json-validator?color=21bb42&label=%F0%9F%93%A6%20npm" /></a>
+	<a href="https://codecov.io/gh/JoshuaKGoldberg/package-json-validator" target="_blank"><img alt="🧪 Coverage" src="https://img.shields.io/codecov/c/github/JoshuaKGoldberg/package-json-validator?label=%F0%9F%A7%AA%20coverage" /></a>
+	<a href="https://github.com/JoshuaKGoldberg/package-json-validator/blob/main/LICENSE.md" target="_blank"><img alt="📝 License: MIT" src="https://img.shields.io/badge/%F0%9F%93%9D_license-MIT-21bb42.svg" /></a>
+	<a href="http://npmjs.com/package/package-json-validator" target="_blank"><img alt="📦 npm version" src="https://img.shields.io/npm/v/package-json-validator?color=21bb42&label=%F0%9F%93%A6%20npm" /></a>
+	<img alt="💪 TypeScript: Strict" src="https://img.shields.io/badge/%F0%9F%92%AA_typescript-strict-21bb42.svg" />
 </p>
 
-## Supported Specifications
-
-Of course, there are multiple ones to follow, which makes it trickier.
-
-- [NPM](https://docs.npmjs.com/cli/configuring-npm/package-json)
-- [CommonJS Packages 1.0](http://wiki.commonjs.org/wiki/Packages/1.0)
-- [CommonJS Packages 1.1](http://wiki.commonjs.org/wiki/Packages/1.1)
-
```

Comment:
🐛 This should have been preserved.

Reply:
I ended up moving it down to after `## Usage`. I don't think it needs to be so prominently mentioned anymore.